db/seeds.rb

```diff
@@ -8,5 +8,6 @@
 
 unless User.exists?(admin: true)
   User.create!(email: ENV.fetch("ADMIN_EMAIL"),
-    password: ENV.fetch("ADMIN_PASSWORD"), admin: true)
+    password: ENV.fetch("ADMIN_PASSWORD"), name: ENV.fetch("ADMIN_NAME"),
+    admin: true)
```

Comment:
Having the seeds depend on environment variables interests me. What happens if we forget to set the values? `Hash#fetch` will raise an error, but is the error message clear enough? How would you feel about adding something like the following?

```ruby
unless ENV("ADMIN_PASSWORD").present?
  raise "The database seeds need an `ADMIN_PASSWORD` environment variable"
end
```